### ISSUE TITLE
Is offsets.storage=zookeeper supported?

### ISSUE THREAD

**Issue by ov7a** [fede1024/kafka-view#28]
As far as I can see, old-style zookeeper offsets are not displayed with default settings (I'm using docker image).
Are they supported?

**Comment by fede1024** [comment]
No, they are not supported. Zookeeper offsets require a zookeeper client that kafka-view at the moment doesn't have. I don't have plans to support it since the preferred way to store offsets and groups is now to use Kafka itself.
